Resources/Storage/StorageElement.py

```diff
@@ -253,7 +254,7 @@ def getStatus( self ):
       - DiskCacheTB: float (-1 if not defined)
     """
 
-    self.log.verbose( "StorageElement.getStatus : determining status of %s." % self.name )
+    self.log.getSubLogger( 'getStatus' ).verbose( "determining status of %s." % self.name )
```

Comment:
As these printouts are numerous in DM agents, it is a nightmare to find back what comes from where... Therefore trying to make it easier to debug problems. At the same time it avoids repeating in all logs long and tedious strings. In this specific method it is not the case but some methods had tens of logs